config/initializers/01_vcap_services.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+if ENV["VCAP_SERVICES"].present?
+  Rails.application.config.x.vcap_services = JSON.parse(ENV["VCAP_SERVICES"])
+end
```

Comment:
Do we need this initialiser? Can we just use `ENV["VCAP_SERVICES"]` in the second one?

As a matter of fact, if we move it to the second initialiser, can we get rid of `01_` or `02_` in the initialiser name?